### ISSUE TITLE
Refactor jwt_utils

### ISSUE THREAD

**Issue by k-allagbe** [ai-cfia/membrane-backend#60]
Currently jwt_utils has too many responsabilities. It should focus only on creating, verifying and decodeing JWT tokens.

**Comment by k-allagbe** [comment]
@rngadam as discussed, we could extract the jwt processings that are common to membrane-client into a `common` folder. I could offload the server specific responsibilities to another module and push jwt_util into `common`. Thoughts?

**Comment by rngadam** [comment]
yes, but actually when I said a common sub-module, I didn't necessarily mean literally a sub-module named common. I think it would make more sense to have a membrane.jwt submodule to put common jwt functions that should be present both on the client and server together?
